### ISSUE TITLE
Add Preview Release to MODX Extras Public Database

### ISSUE THREAD

**Issue by benguthrie** [modxcms/a11y#51]
The extra should be available in the public MODX Database.

**Comment by goldsky** [comment]
I would rather to upload the package on this git, because Preview Release is not a stable release.
Developer then could install the package by upload the zip file manually.

**Comment by dubrod** [comment]
I agree Preview Release should probably not be in the Public Database
